JenkinsPipelines/Scripts/JenkinsTestSelectionMenuGenerator.ps1

```diff
@@ -191,6 +191,60 @@ Write-LogInfo "Validating TestByTestnameDetailed.txt..."
 (Get-Content "$DestinationPath\TestByTestnameDetailed.txt") | Where-Object {$_.trim() -ne "" } | Set-Content "$DestinationPath\TestByTestnameDetailed.txt"
 Write-LogInfo "Done"
 
+
+# This file is created for job: pipeline-cloudtest-manual
+# Parameter Name : CATEGORY_AREA
+$AzureCategoryAreas =  "CategoryArea="
+foreach ( $platform in $Platforms )
+{
+    if ($platform -ne "Azure") {
+        continue;
+    }
+    $CurrentCategories = ($xmlData.test | Where-Object { $_.Platform.Contains($platform) }).Category | Sort-Object | Get-Unique
+    foreach ( $category in $CurrentCategories)
+    {
+        $CurrentAreas = ($xmlData.test | Where-Object { $_.Platform.Contains($platform) } | Where-Object { $_.Category -eq "$category" }).Area | Sort-Object | Get-Unique
+        foreach ($area in $CurrentAreas)
+        {
+            if ($category -and $area) {
+                $AzureCategoryAreas += "$category $area,"
+            }
+        }
+    }
+}
+$AzureCategoryAreas = $AzureCategoryAreas.Trim(",")
+$FilePath = "$DestinationPath\Azure-LISAv2-TestCategoryAreas.txt"
```

Comment:
How often is this module exercised? Is it going to be exercised and refresh to new file every time the pipeline Build with parameter page open?

Reply:
No. The data will be refreshed daily.

Reply:
Ok, there will be risk getting the time gap, but we can fix it later, if it gets serious.